DESCRIPTION

```diff
@@ -1,5 +1,5 @@
 Package: r2dii.analysis
-Title: Tools to Measure Climate Scenario Alignment of Corporate Lending
+Title: Tools to Measure Climate Scenario Alignment of Corporate Loans
```

Comment:
Nit: I think it's best to remove "Tools to" as there is a record that CRAN considered that uneccessary (as every package provides tools to do something).

https://github.com/DavisVaughan/extrachecks under _Your package DESCRIPTION Title is flagged._:

> I’ve been flagged for a “redundant” title. I had: “A Toolkit for the Construction of Modeling Packages” which was flagged since “Toolkit for” seemed redundant. I changed it to “Construct Modeling Packages” and was accepted.